data/functions/add-cars-to-aggregate.js

```diff
@@ -0,0 +1,66 @@
+import * as Sentry from '@sentry/serverless'
+import { unmarshall } from '@aws-sdk/util-dynamodb'
+
+import { addCarsToAggregate } from '../index.js'
+import { parseDynamoDbEvent } from '../utils/parse-dynamodb-event.js'
+
+Sentry.AWSLambda.init({
+  environment: process.env.SST_STAGE,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+})
+
+const AWS_REGION = process.env.AWS_REGION || 'us-west-2'
```

Comment:
it will always be set. This is because of TS

We used this pattern in w3infra to separate ENV vars that are set automatically and ours. But I can move it together to not require a fallback value to appease TS

Reply:
```js
```

Reply:
Cannot we use `mustGetEnv` to require value and appease TS?

<img width="335" alt="Screenshot 2023-02-03 at 09 57 45" src="https://user-images.githubusercontent.com/152863/216570491-3436a2ca-b46a-44cb-9350-31532a4a8ad7.png">
<img width="325" alt="Screenshot 2023-02-03 at 09 58 09" src="https://user-images.githubusercontent.com/152863/216570503-aa25bd99-8913-419b-91ff-c5aa2a3ee0ec.png">